app/views/games/_gametable.html.erb

```diff
@@ -1,9 +1,20 @@
-<table class="table table table-striped table-hover table-sm">
-  <thead class="table-dark">
+
+
+<table class="table table table-striped table-hover tabletable-sm">
+  <thead class="table-dark table-bordered ">
     <tr style="text-align: center">
-      <th scope="col">Number</th>
-      <th scope="col">State</th>
-      <th scope="col">Created</th>
+      <th>
+        <%= link_to "Game Number", "?sort=id&order=#{params[:sort] == 'id' && params[:order] == 'asc' ? 'desc' : 'asc'}", class:'plain-link' %>
```

Comment:
`"?sort=id&order=#{params[:sort] == 'id' && params[:order] == 'asc' ? 'desc' : 'asc'}"` This part looks a bit hard to understand. I'd rather use hashes to pass the params, as
```erb
        <%= link_to "State", {
          sort: 'state',
          sort_dir: @sort == 'state' && @sort_dir == 'asc' ? 'desc' : 'asc'
        }, class:'plain-link' %>
```